certified-connectors/personr/personr.swagger.json

```diff
@@ -0,0 +1,487 @@
+{
+  "swagger": "2.0",
```

Comment:
Please modify swagger filename as per the templates mentioned in this path. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json